chef_master/source/backend_failure_recovery.rst

```diff
@@ -153,3 +153,35 @@ Once you've resolved the issue and can run the ``pgsql-follow`` command successf
 
    rm /var/opt/chef-backend/leaderl/data/unsynced
    chef-backend-ctl start leaderl
+
+General Follower Recovery Process
+=====================================
+
+Initial attempts to recover should follow this general pattern and use the scenarios and tools shown above to assist in the recovery steps:
```

Comment:
Was all of your added content - beyond the intro sentence - supposed to be in a numbered list format? (I formatted it as such from line 178 ["In a Chef Backend High Availability deployment..."] onward to be included as content with #4.)

Reply:
Yes, I believe the numbered list is a good way to present it.